app/Keycloak/KeycloakServiceProvider.php

```diff
@@ -15,6 +17,7 @@ protected function getProvidedServiceNames(): array
     {
         return [
             KeycloakUserIdentityResolver::class,
+            CachedUserIdentityResolver::class,
```

Comment:
Don't know if I should work with a featureflag since Redis is already available & used on `udb3-backend` or directly move inline the constructor of CachedUserIdentityResolver?

Reply:
Good for me to move it inline.